stl.go

```diff
@@ -677,6 +681,19 @@ func newTTIBlock(i *Item, idx int) (t *ttiBlock) {
 		verticalPosition:     stlVerticalPositionFromStyle(i.InlineStyle),
 	}
 
+	if i.InlineStyle != nil && i.InlineStyle.STLJustification != nil {
```

Comment:
Could you create a `stlJustificationCodeFromStyle` method with this logic inside instead? A bit like `stlVerticalPositionFromStyle()`